content/en/docs/tasks/traffic-management/request-routing/index.md

```diff
@@ -15,9 +15,10 @@ microservice.
 ## Before you begin
 
 * Setup Istio by following the instructions in the
-[Installation guide](/docs/setup/).
+  [Installation guide](/docs/setup/).
 
-* Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+* Deploy the [Bookinfo](/docs/examples/bookinfo/) sample application including the
+  [default destination rules](/docs/examples/bookinfo/#apply-default-destination-rules).
```

Comment:
This seems unnecesary since there's already a warning on line 43 that the destinations rules are needed.

Reply:
Thanks.